packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -158,7 +158,7 @@ export class SegmentedControl
     valueMissing: false,
   };
 
-  /** Specifies the width of the component. */
+  /** Specifies the width of the component. [Deprecated] The `"half"` value is deprecated, use `"full"` instead. */
```

Comment:
Could you check me on the documentation here @jcfranco. Since we don't we don't have an established pattern with deprecated props, I followed a similar example with shell-panel displayMode for float, as recommended by @geospatialem.